src/ventus/wm/windowmanager.js

```diff
@@ -199,12 +217,12 @@ function($, Window, View, DefaultMode, ExposeMode, FullscreenMode) {
 
 	WindowManager.prototype.createWindow.fromQuery = function(selector, options) {
 		options.content = View(selector);
-		return this.createWindow(options);
+		return this.createWindow(this, options);
```

Comment:
This line and line 220 introduce a new bug, it's sending a new argument but `createWindow` is receiving just one: `options`.

Actually this is unnecessary since you can use `this` inside `createWindow` there's no need to send this new argument.